src/i18n/locales/en_US/video.json

```diff
@@ -10,6 +10,8 @@
   "episode": "episode",
   "episode_not_found": "Episode not found",
   "episodes": "Episodes",
+  "other_episodes": "Other episodes",
+  "other_season_episodes": "Other episodes of the season",
```

Comment:
I don't see that these are used anywhere

Reply:
Yeah, forgot to remove them.